com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingValidation.cs

```diff
@@ -0,0 +1,229 @@
+
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering
+{
+    public class DebugDisplaySettingsValidation : IDebugDisplaySettingsData
```

Comment:
Does this class needs to be public? If so it needs to be fully documented with API docs. Same for all public classes being added.

Reply:
This class doesn't exist on my branch, its widgets were redistributed under DebugDisplaySettingsLighting and DebugDisplaySettingsRendering. I'll make a note to check if they can be made internal.

Reply:
https://jira.unity3d.com/browse/XPIPELINE-155